jobs/job_options.py

```diff
@@ -63,7 +63,6 @@ def __init__(
             region=feconf.GOOGLE_APP_ENGINE_REGION,
             temp_location=feconf.DATAFLOW_TEMP_LOCATION,
             staging_location=feconf.DATAFLOW_STAGING_LOCATION,
-            save_main_session=True,
```

Comment:
I removed this because it interfered with my testing, and because it seems unnecessary. We can include it again later (as we investigate pip packaging and docker images).